design-proposals/live-migration-for-bridged-pod-network.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+This feature enables support of live-migrate for VMs with pod network in bridge and macvtap modes.
+
+## Motivation
+Masquerade and slirp are not so performant compared to bridge and macvtap modes.
+We want to use the less latency modes with the opportunity to live-migrate the virtual machines.
+
+## Goals
+Provide a live-migration feature for VMs running with pod netowork connected in bridged mode and macvtap mode.
+
+## Non Goals
+The live-migration of virtual machines between the pods with different MAC address will invoke NIC reattaching procedure. This might affect some applications which are binding to the specific interface inside the VM.
```

Comment:
Are these assumptions and limitations?  They don't seem like Non-goals. 

Macvtap has the limitation on the [website](https://kubevirt.io/user-guide/virtual_machines/interfaces_and_networks/#macvtap): "Live migration is not seamless, see [issue #5912](https://github.com/kubevirt/kubevirt/issues/5912#issuecomment-888938920)".    You probably want something similar.

Reply:
Thanks, added to the Non goals